iceberg/iceberg-rest-server/src/main/java/org/apache/gravitino/iceberg/service/rest/IcebergTableOperations.java

```diff
@@ -296,7 +297,14 @@ private LoadTableResponse injectCredentialConfig(
     }
     Credential credential =
         CredentialUtils.vendCredential(
-            credentialProvider, loadTableResponse.tableMetadata().location());
+            credentialProvider,
+            new String[] {
+              loadTableResponse.tableMetadata().location(),
+              loadTableResponse.tableMetadata().property(TableProperties.WRITE_DATA_LOCATION, ""),
```

Comment:
it's odd to pass an empty path if the specific properties is not set, could you add a check before adding the extra paths?

Reply:
Updated